lib/docraptor/api_client.rb

```diff
@@ -15,9 +15,11 @@ class ApiClient
     # @return [Hash]
     attr_accessor :default_headers
 
+    # Initializes the ApiClient
+    # @option config [Configuration] Configuration for initializing the object, default to Configuration.default
     def initialize(config = Configuration.default)
       @config = config
-      @user_agent = "ruby-swagger-#{VERSION}"
+      @user_agent = "Swagger-Codegen/#{VERSION}/ruby"
```

Comment:
I thought about that. Think we should fix this? I suspect it will be post language and maybe an upstream to get agent name into config.